src/coreclr/vm/codeman.h

```diff
@@ -1499,6 +1499,14 @@ class NativeUnwindInfoLookupTable
                                          int EndIndex);
 };
 
+class HotColdMappingLookupTable
+{
+public:
+    static int LookupMappingForMethod(PTR_ULONG pLookupTable,
```

Comment:
Can we have a comment saying what this function is supposed to return?

Reply:
Sure thing.